src/components/comment-input.component.js

```diff
@@ -90,6 +92,13 @@ export class CommentInput extends Component {
 
     return (
       <View style={styles.container}>
+        <MentionArea
+          trigger="@"
+          text={this.state.text}
+          updateText={text => this.setState({ text })}
```

Comment:
Long story short, it hurts performance. You will find more info here =] https://stackoverflow.com/a/36677798